src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/LinkDefendantToClaimTest.java

```diff
@@ -30,7 +30,8 @@
     properties = {
         "core_case_data.api.url=false",
         "feature_toggles.ccd_async_enabled=false",
-        "feature_toggles.ccd_enabled=false"
+        "feature_toggles.ccd_enabled=false",
+        "feature_toggles.reminderEmails=true"
```

Comment:
Please use `feature_toggles.reminder_emails` to be consistent with other flags.

Reply:
@sabahirfan There is another `feature_toggle`: `emailToStaff` already so I copied that style as it's related. Happy to change still, and happy to refactor the existing `emailToStaff`. Generally though, there is no consistency in case strategy in `application.yml`.

Please confirm request.

Reply:
Missed that flag (`emailToStaff`). Would be better if can be consistent. Leave that with you.